includes/output.inc

```diff
@@ -150,10 +150,14 @@ function dt($string, $args = array()) {
   // The language system requires a working container which has the string
   // translation service.
   else if (drush_drupal_major_version() >= 8 && \Drupal::hasService('string_translation')) {
-    // Drupal 8 deprecates (and will eventually remove) !var replacements,
-    // so we'll pre-replace these before calling translate().
+    // Drupal 8 removes !var replacements, creating a user-level error when
+    // these are used, so we'll pre-replace these before calling translate().
     $legacy_args = find_legacy_dt_args($args);
     $string = strtr($string, $legacy_args);
+    // Remove the legacy replacements.
+    foreach (array_keys($legacy_args) as $legacy_arg_key) {
```

Comment:
Convert them into? strtr() inside of the dt() call in the 100+ instances where we use !var rather than doing it in find_legacy_dt_args()?

Not opposed to that but also wondering what we gain by doing so? In drush we shouldn't be having the usual sanitization concerns because the output is CLI